src/com/spikes2212/command/genericsubsystem/InjectedSubsystem.java

```diff
@@ -0,0 +1,8 @@
+package com.spikes2212.command.genericsubsystem;
+
+import com.google.inject.Injector;
+
+public abstract class InjectedSubsystem extends GenericSubsystem {
+	private Injector injector;
+	
```

Comment:
please add an `injectCommand` method. that receives a commnad, uses `Injector.injectMembers` on it and then returns it.